sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -232,7 +234,9 @@ object BasicWriteJobStatsTracker {
       NUM_FILES_KEY -> SQLMetrics.createMetric(sparkContext, "number of written files"),
       NUM_OUTPUT_BYTES_KEY -> SQLMetrics.createSizeMetric(sparkContext, "written output"),
       NUM_OUTPUT_ROWS_KEY -> SQLMetrics.createMetric(sparkContext, "number of output rows"),
-      NUM_PARTS_KEY -> SQLMetrics.createMetric(sparkContext, "number of dynamic part")
+      NUM_PARTS_KEY -> SQLMetrics.createMetric(sparkContext, "number of dynamic part"),
+      DURATION_JOB_COMMIT->
+        SQLMetrics.createTimingMetric(sparkContext, "duration of committing the job")
```

Comment:
Can you add some tests somewhere, e.g., `SQLMetricsSuite`?

Reply:
> Can you add some tests somewhere, e.g., `SQLMetricsSuite`?

Yea, UT added in `SQLMetricsSuite`